examples/seismic/viscoelastic/viscoelastic_example.py

```diff
@@ -41,17 +41,17 @@ def run(shape=(50, 50), spacing=(20.0, 20.0), tn=1000.0,
                                 space_order=space_order, constant=constant, **kwargs)
     info("Applying Forward")
     # Define receiver geometry (spread across x, just below surface)
-    rec1, rec2, vx, vz, txx, tzz, txz, summary = solver.forward(autotune=autotune)
+    rec1, rec2, v, tau, summary = solver.forward(autotune=autotune)
 
     return (summary.gflopss, summary.oi, summary.timings,
-            [rec1, rec2, vx, vz, txx, tzz, txz])
+            [rec1, rec2, v, tau])
 
 
 def test_viscoelastic():
-    _, _, _, [rec1, rec2, vx, vz, txx, tzz, txz] = run()
+    _, _, _, [rec1, rec2, v, tau] = run()
     norm = lambda x: np.linalg.norm(x.data.reshape(-1))
-    assert np.isclose(norm(rec1), 5.6222677, atol=1e-3, rtol=0)
-    assert np.isclose(norm(rec2), 0.26486522, atol=1e-3, rtol=0)
+    assert np.isclose(norm(rec1), 5.774392, atol=1e-3, rtol=0)
```

Comment:
remind me why this change (@rhodrin )

Reply:
From a tweak to the critical `dt` in viscoelastic model. A 6./7. has been changed to 0.85.